lib/ffmpeg/encoderbase.py

```diff
@@ -59,6 +59,7 @@ def hwformat(self):
   forced_idr    = property(lambda s: s.ifprop("vforced_idr", " -forced_idr 1 -force_key_frames expr:1"))
   maxframesize  = property(lambda s: s.ifprop("maxframesize", " -max_frame_size {maxframesize}k"))
   pict          = property(lambda s: s.ifprop("vpict", " -pic_timing_sei 0"))
+  roi           = property(lambda s: s.ifprop("vroi", ",addroi=0:0:{width}/2:{height}/2:-1/3"))
```

Comment:
We don't need to prefix property names with `v` anymore because we wrap them with `variants` key in the test config now.  Please remove the prefix (`vroi` -> `roi`).